orcid-core/src/main/resources/org/orcid/core/template/digest_email_html.ftl

```diff
@@ -82,6 +82,7 @@
                     </tbody>					                      
 	        	</table>
 	        	</#list> 
+>>>>>>> master
```

Comment:
@TomDemeranville how did this get in here :-) There shouldn't be changes in this section.